packages/app/src/ui/pages/Onboarding/Onboarding.tsx

```diff
@@ -1,26 +1,36 @@
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
+
 import logoSVG from "@src/static/icons/logo.svg";
-import { ButtonType, Button } from "@src/ui/components/Button";
 import { Icon } from "@src/ui/components/Icon";
 import { PasswordInput } from "@src/ui/components/PasswordInput";
 
 import "./onboarding.scss";
 import { useOnboarding } from "./useOnboarding";
 
 const Onboarding = (): JSX.Element => {
-  const { errors, isLoading, register, onSubmit, isShowPassword, onShowPassword } = useOnboarding();
+  const { errors, isLoading, register, onSubmit, isShowPassword, onShowPassword, onGoToOnboardingBackup } =
+    useOnboarding();
 
   return (
-    <form className="flex flex-col flex-nowrap h-full onboarding" data-testid="onboarding-form" onSubmit={onSubmit}>
-      <div className="flex flex-col items-center flex-grow p-8 onboarding__content">
+    <Box
+      className="onboarding"
+      component="form"
+      data-testid="onboarding-form"
+      sx={{ display: "flex", flexDirection: "column", flexWrap: "nowrap", height: "100%" }}
+      onSubmit={onSubmit}
+    >
+      <Box sx={{ display: "flex", flexDirection: "column", alignItems: "center", flexGrow: 1, p: 3 }}>
         <Icon size={8} url={logoSVG} />
 
-        <div className="text-lg pt-8">
-          <b>Thanks for using CryptKeeper!</b>
-        </div>
+        <Typography sx={{ pt: 3, fontWeight: "bold" }} variant="h4">
```

Comment:
I think this change what makes this page looks bigger in my case with this scroll bar.  
![Screenshot_20230728_152630](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/36774944/fec3be00-716a-4f32-848f-53f692682928)